docs/sources/datasources/cloudmonitoring.md

```diff
@@ -226,6 +226,24 @@ The Alias By field allows you to control the format of the legend keys for SLO q
 
 SLO queries use the same [alignment period functionality as metric queries]({{< relref "#metric-queries" >}}).
 
+### MQL (Monitoring Query Language) queries
+
+> Only available in Grafana v7.x+
+
+The MQL query builder in the Google Cloud Monitoring data source allows you to display MQL result in time series format. To get an understanding of the basic concepts in MQL, please refer to Google Cloud Monitoring's [official docs](https://cloud.google.com/monitoring/mql).
+
+#### How to create an MQL query
+
+To create an MQL query, follow these steps:
+
+1. Choose the option **Monitoring Query Language (MQL)** in the **Query Type** dropdown.
+1. Choose a project from the **Project** dropdown.
+1. Specify [MQL](https://cloud.google.com/monitoring/mql/query-language) to query field.
```

Comment:
Unclear. Am I selecting MQL from a list? Am I typing the entry somewhere? If so, where? 

Also, the link here is confusing. Why are you sending me to another website?

Reply:
User edit MQL in text area.
![image](https://user-images.githubusercontent.com/224552/97247497-25bdc000-1843-11eb-8f73-c937b3f479db.png)

I follow the convention to include related link in step guide.